src/mpi/romio/adio/ad_ime/ad_ime_delete.c

```diff
@@ -0,0 +1,33 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil ; -*- */
+/*
+ *
+ *   Copyright (C) 1997 University of Chicago.
+ *   Copyright (C) 2017 DataDirect Networks.
+ *   See COPYRIGHT notice in top-level directory.
+ */
+
+#include "ad_ime.h"
+#include "adio.h"
+
+#include "ad_ime_common.h"
+
+void ADIOI_IME_Delete(const char *filename, int *error_code)
+{
+    int ret;
+    static char myname[] = "ADIOI_IME_DELETE";
+
+    char *ime_filename = ADIOI_IME_Add_prefix(filename);
+    ret = ime_native_unlink(ime_filename);
+    ADIOI_Free(ime_filename);
+    if (ret)
+        *error_code = MPIO_Err_create_code(MPI_SUCCESS,
+                 MPIR_ERR_RECOVERABLE,
+                 myname, __LINE__,
+                 ADIOI_IME_error_convert(ret),
```

Comment:
Where is `ADIOI_IME_error_convert` defined?  Did this patch forget a file?

Reply:
Thanks for reviewing the patches! 
I've figured out that few changes were missing in the patch (I've probably forgotten to add the changes to the git commit). Thanks for noticing this! 
I'm going to submit a new version of the patch in a few minutes.